dash-pipeline/bmv2/dash_metadata.p4

```diff
@@ -134,14 +46,6 @@ enum bit<8> dash_eni_mac_type_t {
 struct conntrack_data_t {
     bool allow_in;
     bool allow_out;
```

Comment:
the following fields are flow related and intentionally being aggregated into this structure, because conntrack_data_t is used to hold all data that is related to flow.

and many of them are moved out from this structure and some fields seem to be missing, such as reverse_flow_key (it somehow doesn't show up in my search).

it doesn't feel to be right to me. this structure should not be changed. I can understand moving some of the data types as header, but the metadata should be remained - as they are different from header conceptually.

Reply:
For flow, I intend to use a new flow metadata.